src/test/java/uk/gov/companieshouse/api/accounts/controller/CompanyAccountControllerTest.java

```
@@ -39,13 +43,22 @@
 
     @BeforeEach
     public void setUp(){
-        when(companyAccountService.save(companyAccount)).thenReturn(createdCompanyAccount);
+        try {
+            when(companyAccountService.save(any(CompanyAccount.class), anyString())).thenReturn(createdCompanyAccount);
+        } catch (NoSuchAlgorithmException e) {
+            e.printStackTrace();
+        }
     }
 
     @Test
     @DisplayName("Tests the successful creation of an companyAccount resource")
     public void canCreateAccount() {
-        ResponseEntity response = companyAccountController.createCompanyAccount(companyAccount);
+        ResponseEntity response = null;
+        try {
+            response = companyAccountController.createCompanyAccount(companyAccount);
+        } catch (NoSuchAlgorithmException e) {
+            e.printStackTrace();
```

The test should fail if an exception is unexpectedly thrown